hwilib/commands.py

```diff
@@ -53,41 +53,12 @@ def get_client(device_type, device_path, password=None):
 # Get a list of all available hardware wallets
 def enumerate(args):
```

Comment:
It would be nice if `enumerate` didn't ignore `--device-type=` as it's a bit slow.

Reply:
I like the suggestion but obviously should wait until this is merged